### ISSUE TITLE
Error: It seems there is already an App at '/Applications/<name>.app when setting "homebrew_cask_accept_external_apps" option to true

### ISSUE THREAD

**Issue by xserrat** [geerlingguy/ansible-role-homebrew#114]
Hi @geerlingguy,

I've just realised that the option `homebrew_cask_accept_external_apps: true` does not work. I'm using `MacOS 10.14 Mojave` and `Ansible 2.5`.

I've just commented in a closed issue related to this problem:

https://github.com/ansible/ansible/issues/33975#issuecomment-445564355

Thank you so much :)

**Comment by elliotweiser** [comment]
Just ran this with Ansible 2.5.13. I have Steam installed on my Mac outside of Homebrew.

```
$ ansible localhost -i inventory -m homebrew_cask -a 'name=steam state=present accept_external_apps=true'
localhost | SUCCESS => {
    "changed": false,
    "msg": "Cask already installed: steam"
}
$ brew cask list | grep steam
$
```

It also works as expected on Ansible 2.7. You must be using an outdated version of Ansible.

**Comment by xserrat** [comment]
Thanks @elliotweiser , everything is ok. The error is that I was executing the ansible role `ansible-role-homebrew` with the option `homebrew_cask_accept_external_apps` in version 2.6 and this option is available in version 2.7.
